### PR TITLE
Add more svg css properties

### PR DESCRIPTION
**Pull request by AgtLucas**
@chenglou Just pushed more properties here. :)

### CODE REVIEW

**Comment by chenglou** [comment]
kk. Are they complete now? Btw what about https://github.com/reasonml/reason-react/issues/14

**Comment by AgtLucas** [comment]
@chenglou AFAIK is complete now, but we can have a second review.

Which part about the #14?

**Comment by chenglou** [comment]
I'm not too familiar with the pointerEvent. I thought this was related?

File: src/reactDOMRe.re [hunk 1]
```diff
@@ -913,13 +913,33 @@ module Style = {
     rotationPoint::string? =>
 
     /* svg */
```
Comment (glennsl) [inline src/reactDOMRe.re]: Could you change this to `SVG 1.1 - REC`?

**Comment by glennsl** [comment]
`pointerEvents` was added in #15, so #14 should be good now.

**Comment by AgtLucas** [comment]
@glennsl Done!

**Comment by chenglou** [comment]
Great! Thanks =D

**Pull request merged**